dear_petition/petition/export/mapper.py

```diff
@@ -44,157 +49,34 @@ def map_petitioner(data, petition, extra={}):
 
 
 def map_attorney(data, petition, extra={}):
-    pass
+    attorney = extra.get("attorney", None)
+    data["NameAtty"] = getattr(attorney, "name", None)
```

Comment:
Attorney attributes are allowed to be ``blank`` on the model, so I think the default empty value of an empty string should be fine for these, e.g. ``attorney.address1``.

Reply:
done